driver/src/main/java/eu/cloudnetservice/cloudnet/driver/permission/Permission.java

```diff
@@ -53,25 +93,59 @@ public int compareTo(@NonNull Permission o) {
     private int potency = 0;
     private long timeOutMillis = 0;
 
+    /**
+     * Sets the name of this permission.
+     *
+     * @param name the name of the permission.
+     * @return the same instance for chaining.
```

Comment:
```suggestion
     * @return the same instance as used to call the method, for chaining.
```
To stay consistent **This applies to all other builder methods as well!**